push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/GCMPushNotificationSender.java

```diff
@@ -81,6 +81,11 @@ public void sendPushMessage(Variant variant, Collection<String> tokens, UnifiedP
         gcmBuilder.addData("alert", message.getAlert());
         gcmBuilder.addData("sound", message.getSound());
         gcmBuilder.addData("badge", "" + message.getBadge());
+        /*
+        The Message defaults to a Normal priority.  High priority is used
+        by GCM to wake up devices in Doze mode as well as apps in AppStandby 
+        mode.  This has no effect on devices older than Android 6.0
+        */
```

Comment:
cool!

Feel free to merge to master!

On Monday, 28 March 2016, Hoyt Summers Pittman <notifications@github.com>
wrote:

> In
> push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/GCMPushNotificationSender.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/701#discussion_r57570738>
> :
>
> > @@ -81,6 +81,11 @@ public void sendPushMessage(Variant variant, Collection<String> tokens, UnifiedP
> >          gcmBuilder.addData("alert", message.getAlert());
> >          gcmBuilder.addData("sound", message.getSound());
> >          gcmBuilder.addData("badge", "" + message.getBadge());
> > +        /*
> > +        The Message defaults to a Normal priority.  High priority is used
> > +        by GCM to wake up devices in Doze mode as well as apps in AppStandby
> > +        mode.  This has no effect on devices older than Android 6.0
> > +        */
>
> I actually can't find it documented anywhere that this is Google's policy
> so it might be something I just read on Reddit.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/701/files/b495835292a5ccfaeae32c45743a21d2dd49f9cd..822a84a815e0f372d33b248cdd43389579b46a15#r57570738>
>


-- 
Sent from Gmail Mobile